runtime/gc_glue_java/ArrayletObjectModel.hpp

```diff
@@ -852,6 +876,11 @@ class GC_ArrayletObjectModel : public GC_ArrayletObjectModelBase
 	 * Tear down the receiver
 	 */
 	void tearDown(MM_GCExtensionsBase *extensions);
+
+	/**
+	 * Assert MM_unreachable if the element size is invalid
+	 */
+	void AssertBadElementSize();
```

Comment:
Any name recommendations for this function?

Reply:
Move function to the private section (close to line 56). Also comment is not relevant